index.html

```diff
@@ -0,0 +1,277 @@
+              <p class="s7p color-2"></p>
+            </div>
+          </article>
+        </div>
+    </section>
+
+    <section class="section-eight color-3">
+      <article class="s82">
+          <h1 class="s6h1 color-1"></h1>
+          <h3 class="s6h3 color-1"></h3>
+          <p class="s8p color-2"></p>
+          <button class="s6btn color-1" type="button"></button>
+      </article>
+    
+    </section>
+
+    <section class="section-nine">
+      <div class="grid-div">
+        <article class="s9">
+          <article class="s82">
+            <h1 class="s6h1 color-1"></h1>
+            <h3 class="s9h3 color-1"></h3>
+            <p class="s8p color-2"></p>
+            <button class="s6btn color-1" type="button"></button>
+        </article>
+        </article>
+        <article class="s9">
+          <article class="s82">
+            <h1 class="s6h1 color-1"></h1>
+            <h3 class="s9h3 color-1"></h3>
+            <p class="s8p color-2"></p>
+            <button class="s6btn color-1" type="button"></button>
+        </article>
+        </article>
+        <article class="s9">
+          <article class="s82">
+            <h1 class="s6h1 color-1"></h1>
+            <h3 class="s9h3 color-1"></h3>
+            <p class="s8p color-2"></p>
+            <button class="s6btn color-1" type="button"></button>
+        </article>
+        </article>
+        <article class="s9">
+          <article class="s82">
+            <h1 class="s6h1 color-1"></h1>
+            <h3 class="s9h3 color-1"></h3>
+            <p class="s8p color-2"></p>
+            <button class="s6btn color-1" type="button"></button>
+        </article>
+        </article>
+      </div>
+    </section>
+
+    <footer class="main-footer footer-div center color-2">
+      <div class="links center color-1">
+        <a class="an color-1" href="#">link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link</a>
+        <a class="an color-1" href="#">link,link,link,link,link,link,link,link,link,link,link,link,link,link,link</a>
+        <a class="an color-1" href="#">link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link</a>
+      </div>
+      <div class="links center color-1">
+        <a class="an color-1" href="#">linklink,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link</a>
+        <a class="an color-1" href="#">link, link link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link</a>
+      </div>
+      <div class="links-b center color-1">
+        <a class="an color-1" href="#">linklink,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link</a>
+        <a class="an color-1" href="#">link, link link,link,link,link,link,link,link,link,link,link,link,link,link,link,link,link</a>
+      </div>
```

Comment:
- [ ] Great job centering your links. Can you please add more links instead of putting large strings of link,link,link...? The point is to have different elements that will be positioned according to your rules. I'm sure you can do it.